website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -1473,6 +1509,8 @@ status:
       - arm64
 ```
 
+Note that Karpenter can only discover architecture based requirements. The one exception is using alias, in which case it can discover additional requirements (e.g. GPU / Accelerator support).
```

Comment:
Did we need to say this twice? I see that we called this out above

Reply:
I'm good with dropping it, you're right that it's duplicated.